scout/server/blueprints/variant/views.py

```diff
@@ -281,6 +282,32 @@ def evaluation(evaluation_id):
             variant_id=evaluation_obj["variant_specific"],
         )
         store.delete_evaluation(evaluation_obj)
+
+        # check if this was the last acmg evaluation left on the variant
```

Comment:
This works fine, but perhaps it would be clearer if the evaluation reset was handled by a dedicated method in the controllers? This way would also be easily testable.. ;)